spec/models/contacts_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  fixtures :contacts
+  fixtures :users
+
+  context 'name is not empty' do
+    it 'return a count of 3 after saving a new entry in the contact table' do
```

Comment:
The verb in the message should be in the third-person form, so the example and the contexts containing it can be read as a "normal" English sentence:
- When **the name is empty** it **creates a new record**